content/cybersecurity/concepts/nmap/terms/udp-scan/udp-scan.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: 'UDP Scan'
+Description: 'Scans (only) the UDP Ports of a target, which are not scanned in a regular scan. Can be combined with any TCP scan'
+Subjects:
+  - 'Computer Science'
+  - 'Networking'
+Tags:
+  - 'Networking'
+  - 'Cybersecurity'
+  - 'Ethical Hacking'
+CatalogContent:
+  - 'introduction-to-cybersecurity'
+  - 'paths/scan-systems-with-nmap'
+---
+
+An **UDP Scan** scans (only) the UDP Ports of a target, while the [regular scan](https://www.codecademy.com/resources/docs/cybersecurity/nmap/nmap-scan) only scans the TCP Ports. It is performed by using the `-sU` option and can be combined with any TCP scan type, for example SYN scan (`-sS`) to scan UDP and TCP Ports in the same run. UDP Scan is normally slower and more difficult than a TCP scan.
+
+## Syntax
+
+```pseudo
+nmap -sU <target>
+```
+
+The above conducts an UDP scan on `<target>`.
```

Comment:
```suggestion
The `-sU` option is used to perform a UDP scan on a `<target>`. It can be combined with any TCP scan type. For example, a stealth (SYN) scan (`-sS`) checks for UDP and TCP ports in the same run.
```